openapi/openapi.yml

```
@@ -2187,6 +2188,14 @@ components:
       required: false
       schema:
         type: string
+    tags:
+      description: 'Filter by tags in key:value format (comma-separated for multiple tags).'
```

We don't include `.` at the end of other descriptions.

```suggestion
      description: 'Filter by tags in key:value format (comma-separated for multiple tags)'
```